tvb_documentation/demos/simulate_seeg_from_epileptors_with_observation_noise.ipynb

```diff
@@ -325,7 +325,7 @@
     "\n",
     "#Plot raw time series\n",
     "plt.figure(figsize=(10,10))\n",
-    "plt.plot(ttavg[:], tavg[:, 0, :, 0] + np.r_[:84], 'r')\n",
+    "plt.plot(ttavg[:], tavg[:, 0, :, 0] + np.r_[:76], 'r')\n",
```

Comment:
This looks like a conn.nr_regions value, can't we use that variable to have this code more generic than directly hardcode it?

Reply:
I will do a new commit with changes on this notebook very soon.